azure_monitor/tests/test_protocol.py

```diff
@@ -5,23 +5,24 @@
 
 from azure_monitor import protocol
 
+
 class TestProtocol(unittest.TestCase):
-    def test_object(self):
-        data = protocol.BaseObject()
-        self.assertEqual(repr(data), '{}')
-        data.foo = 1
-        self.assertEqual(data.foo, 1)
-        self.assertEqual(data['foo'], 1)
-        data['bar'] = 2
-        self.assertEqual(data.bar, 2)
-        self.assertEqual(data['bar'], 2)
-        self.assertRaises(KeyError, lambda: data['baz'])
-        self.assertRaises(AttributeError, lambda: data.baz)
+    # def test_object(self):
```

Comment:
Please enable tests and update if needed

Reply:
Done